### PR TITLE
cranelift: Port `vselect` over to ISLE on x64

### PR DESCRIPTION
**Pull request by fitzgen**

### CODE REVIEW

**Review by alexcrichton — APPROVED** [review]

**Comment by abrown** [comment]
@fitzgen, just a heads up that I have a "port `select` to ISLE" change in-progress; just letting you know since I've seen some select-related changes recently. Do you want to take over `select`?

**Comment by fitzgen** [comment]
> Do you want to take over `select`?

I don't need to at the moment, I have my hands full with `sshr` now.

File: cranelift/filetests/filetests/runtests/simd-iabs.clif [hunk 1]
```diff
@@ -4,27 +4,6 @@ target aarch64
...
-    return v1
-}
-; run: %iabs_i16x8([0 0 32767 32767 -32767 -32767 -32768 -32768]) == [0 0 32767 32767 32767 32767 -32768 -32768]
-
-function %iabs_i32x4(i32x4) -> i32x4 {
-block0(v0: i32x4):
-    v1 = iabs v0
-    return v1
-}
-; run: %iabs_i32x4([0 2147483647 -2147483647 -2147483648]) == [0 2147483647 2147483647 -2147483648]
```
Comment (bjorn3) [inline cranelift/filetests/filetests/runtests/simd-iabs.clif:26]: Why did you remove these tests?
